.ci/run.sh

```diff
@@ -37,7 +37,7 @@ case "${CI_JOB}" in
 		echo "INFO: Running QAT integration test"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make qat"
 		;;
-	"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CRI_CONTAINERD_K8S_DEVMAPPER"|"CC_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR")
+	"CRI_CONTAINERD"|"CRI_CONTAINERD_K8S"|"CRI_CONTAINERD_K8S_DEVMAPPER"|"CC_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR")
```

Comment:
Hi @stevenhorsman,

My commit 4614118dea50faa095ca5dd246c9ae527aa20ae7 ("ci: run all containerd tests on the CC build") merged in `main` changed the set of tests executed for the CC_* jobs. I thought it was a good idea because we would have the opportunity to run all the tests to check the functionality of our CC branch of containerd.  On the CCv0 branch we already have CI jobs (for example [kata-containers-CCv0-tests-ubuntu-20.04-PR](http://jenkins.katacontainers.io/view/CCv0/job/kata-containers-CCv0-tests-ubuntu-20.04-PR/)) running those non-CC regular tests.

Thus, I suggest that we return to the old behavior, i.e., the CC_* jobs to run CC tests only. In this case, removing all the CC_* entries from this line 40; remove the lines 63-66 and keep 68-71.

Does it make sense?

Reply:
Hey Wainer, just to confirm I understand properly. Are you suggesting we change the CC_* jobs to only run the CC tests in `main` and then merge that in, or leave it as is in `main` and just keep the old functionality in `CCv0`?